### ISSUE TITLE
Add more checks

### ISSUE THREAD

**Issue by sminnee** [silverstripe/moduleratings#16]
For use with #15 

composer.json content checks
 * Check that a php requirement is defined
 * Check that PSR-4 autoloading is configured
 * Check that phpunit is installed as a dev dependency
 * Check that phpcs is installed as a dev dependency
 * Check that type is silverstripe-vendormodule 

file presence checks (this could be one, configurable, check class)
 * Check for a phpcs.xml.dist
 * Check for a phpunit.xml.dist
 * Check for a .travis.yml

**Comment by robbieaverill** [comment]
I think (and hope!) all of these should be pretty easy to implement

**Comment by sminnee** [comment]
Yeah FYI this would be not for addons.silverstripe.org, but more as a tool similar to phpcs but for your whole project rather than just your PHP code.

**Comment by robbieaverill** [comment]
Yeah I assumed that was the case from your other issue. I like it - the `moduleratings-plugin` is the Composer plugin. It would probably need an update for Composer 2.0 at some point though.
